guide/src/whirlwind-tour/basic-usage.md

```diff
@@ -103,6 +103,8 @@ support! Let's take a look at that and see how it works.
 First create an `index.js` file:
 
 ```js
+require('file-loader?name=[name].[ext]!./index.html');
```

Comment:
You can use [html-webpack-plugin](https://github.com/jantimon/html-webpack-plugin/blob/master/README.md) instead

Reply:
Is there any reason to use that instead? By default, it generates its own HTML instead of copying the one in your project (i.e. the one the guide told you to write). That's not what you want if you're planning to interact with the user.

[This](https://github.com/jantimon/html-webpack-plugin/blob/master/README.md#writing-your-own-templates) might work, but it involves learning about a templating language, and overall seems much more involved than dropping a single `require` line.

Reply:
It can also copy the template and if anybody who knows html and templating, they can also playaround with it. 

So templating language is not a must and it is much preferred way to handle html files.

Reply:
It seems much more powerful, but is templating something useful in a "Getting started with `wasm-bindgen`" guide? I went for what seemed to be the simplest solution, in order not to scare away non-`node` devs.

But sure, I'll try to switch to that instead.

Reply:
Done